src/utils.ts

```diff
@@ -0,0 +1,23 @@
+import { DataFrame, MetricFindValue, DataFrameView } from "@grafana/data";
+
+interface SimpleResult {
+    id: string;
+    name: string;
+}
+
+export function frameToMetricFindValues(df: DataFrame): MetricFindValue[] {
+    const res:MetricFindValue[] = [];
+    const view = new DataFrameView<SimpleResult>(df);
+    view.forEach(item => {
+        const id = item.id;
+        const name = item.name;
+        if (id && name) {
+            res.push({text: name, value:id});
+        }
+        const value = id || name;
```

Comment:
should this be inside an else so that we only push either or, instead of both scenarios?

Reply:
it is weird -- in 7.5+, the DataFrameView would let us check fields outside the loop, but I don't think we want to bump the requirements for that.

I think this is OK as a first pass

Reply:
dooh -- yes obviously!!! 🤦 

good thing we are writing tests!!!